docs/blueprints/EP021.rst

```diff
@@ -0,0 +1,183 @@
+:EP: BP021
+:Title: Kytos testing pipeline and definitions
+:Authors:
```

Comment:
@josemauro also contributed to create the specification document, I think he can be an author too :)